src/tdastro/astro_utils/passbands.py

```diff
@@ -115,14 +118,42 @@ class Passband:
         A 2D array of wavelengths and normalized transmissions.
     """
 
-    def __init__(self, survey, label, table_path=None, table_url=None):
+    def __init__(
+        self,
+        survey: str,
+        label: str,
+        table_path: Optional[str] = None,
+        table_url: Optional[str] = None,
+        units: Optional[Literal["nm", "A"]] = "A",
+    ):
+        """Construct a Passband object
+
+        Parameters
+        ----------
+        survey : str
+            Which survey does this passband correspond to. Often "LSST"
+        label : str
+            Which label does this passband correspond to, Sometimes a filter name like "g"
+        table_path : str, optional
+            Path to the table defining the passband on the filesystem. Will take precedence over table_url
+        table_url : str, optional
+            URL to download the table from.
+        units : Literal['nm','A'], optional
```

Comment:
This has to be more general if we decide to include units. Also `wave_units` may be better here.